src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_params.py

```diff
@@ -82,7 +82,7 @@ def load_arguments(self, _):  # pylint: disable=too-many-statements
         c.argument('source_registry_username', options_list=['--username', '-u'], help='The username of source container registry')
         c.argument('source_registry_password', options_list=['--password', '-p'], help='The password of source container registry')
         c.argument('target_tags', arg_type=image_by_tag_type, action='append')
-        c.argument('repository', help='The repository name to do a manifest-only copy for images.', action='append')
+        c.argument('repository', help='The repository name to do a manifest-only copy for images. Multiple --repository can be fed.', action='append')
```

Comment:
help='The repository name for a manifest-only copy of images. Multiple copies supported by passing --repository multiple times.